### DISCUSSION TITLE
BthPS3 Bluetooth Drivers v2.1.153

### DISCUSSION THREAD

**Discussion by nefarius** [nefarius/BthPS3#22] [Announcements]
# v2.1.153 (setup-v2.1.153) BthPS3 changelog

This project was only possible with the investment of many **years** of learning, research and dedication. You're receiving this work for free, if it brought you joy, consider giving back and spend a few coins ❤️

[![DonateEUR](https://user-images.githubusercontent.com/286631/126007336-0b0cfa6d-b0bd-408f-9c1f-46c4227778fb.gif)](https://vigem.org/Donations/)

## How to install

Simply download the setup below and click through it. Done! 

If you want to run it silently, invoke the setup with the `/qn` argument to suppress any user interaction.

If you need to troubleshoot the installation, add `/L*V .\install.log` arguments to generate a detailed log file.

## Remarks

---

⚠️ **This is a new major release which breaks backwards compatibility!** ⚠️

This release **is not compatible with** [Shibari](https://github.com/ViGEm/Shibari) (any version) or [DsHidMini](https://github.com/ViGEm/DsHidMini) version **1**.x.x!

**Do not** install BthPS3 version **2**.x.x if you still want to use these companion solutions, use an older version instead.

--- 

## Bugfixes

- Fixed Device ID, Hardware IDs and Instance ID format for child devices to comply with the majority of input APIs and game engines ([see](https://github.com/ViGEm/DsHidMini/discussions/106))
- Fixed incorrect use of Instance ID causing issues with function drivers (#20)

<hr /><em>This discussion was created from the release <a href='https://github.com/ViGEm/BthPS3/releases/tag/setup-v2.1.153'>BthPS3 Bluetooth Drivers v2.1.153</a>.</em>

**Comment by TSnatch72** [comment]
Do you need to clean uninstall BthPS3 v1.x.x versions in order to upgrade to v2? If so, is there some uninstall guide?

**Reply by nefarius** [reply]
Yes you need to, like all the version before and Setup will even tell you.

You uninstall the previous version like any other piece of software in Programs & Features or Apps or however it's called nowadays in Windows 10.

**Comment by NLPHLL** [comment]
Will this work with no error on Windows 11?
Do I need to disconnect other Bluetooth devices before starting the setup?

**Reply by nefarius** [reply]
Yes, no 😜

**Comment by WettiestMan** [comment]
I have a problem after the installation of the Bluetooth drivers. After installing it, my bluetooth adapter stopped working. It says "This device cannot start (code 10) The specified request is not a valid operation for the target device."

**Reply by nefarius** [reply]
That's an unfixable issue, you can try a different dongle.

**Reply by WettiestMan** [reply]
The thing is, the dongle was working fine, I decided to try it in another PC and worked fine.

**Reply by nefarius** [reply]
I told you all there is to tell, mate.

**Comment by Jurgenson30** [comment]
does it work with a USB bluetooth 2.0 ??   , It does connect to my Bluetooth with SCPtoolkit but is to slow using wireless, and wen I tried  with DsHidMini  via USB works fine  but when I disconnect it and try wireless it shows connected but  on the PS3 controller  the red light are always blinking and never connect to any port, is there any other option or I have to buy a USB Bluetooth  4.0??

**Reply by nefarius** [reply]
Read the README, it's documented there.

**Comment by Adam-Jebrou** [comment]
I get an error when downloading it

![image_2023-01-26_152333396](https://user-images.githubusercontent.com/120840409/214860083-d76d5a53-205e-4fc7-b58c-e400d6cdb091.png)

**Reply by kikecito** [reply]
tengo el mismo error

**Reply by CedyRica** [reply]
I have the same error too after I uninstalled the driver and deleted the driver with driver store explorer.
It worked without problem the first time I installed it.

**Reply by PPancham** [reply]
> I have the same error too after I uninstalled the driver and deleted the driver with driver store explorer. It worked without problem the first time I installed it.

any update guys ?

**Comment by TryEka** [comment]
![nefarius bluetooth](https://github.com/nefarius/BthPS3/assets/60843916/96f2ae14-8b94-46c2-bda6-51d9e933e4d5)

My Nefarius Bluetooth PS Enumerator wont connect, Error Code 10, my bluetooth device is TP Link Bluetooth 5.0 which is compatible with BTHPS3 but still wont connect

Status Device Power Failure
